OpenRA.Game/Traits/World/ActorMap.cs

```diff
@@ -94,25 +94,31 @@ class ProximityTrigger : IDisposable
 
 			readonly Action<Actor> onActorEntered;
 			readonly Action<Actor> onActorExited;
+			readonly bool considerVRange;
 
 			WPos position;
 			WDist range;
+			WDist vRange;
```

Comment:
Can we merge `considerVRange` into this (special case `vRange == 0` to mean any height, since a range of 0 is not meaningful on its own)?